src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -242,6 +246,28 @@ services.AddHttpClient(
 Note: The single instance returned by `HttpClientFactory` is reused by all
 export requests.
 
+## Experimental features
+
+The OTLP exporter contains the following experimental features that are
+available only in pre-release package:
+
+* The [exemplar
+  specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exemplar)
+  is not yet stable. Pre-release builds of the OpenTelemetry .NET SDK and the
+  OTLP exporter support sampling and exporting exemplars.
+* Support for exporting the following fields on `LogRecord`: `CategoryName`,
+  `EventId`, and `Exception`. These fields are unique to OpenTelemetry .NET's
+  data model (i.e., they are not reflected in the [standard data
+  model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#log-and-event-record-definition)).
+  Therefore, the conventions for exporting these fields is not yet stable. In
```

Comment:
```suggestion
  While they can be exported as attributes, there is no established conventions for the attribute key names. In
```

Reply:
While they can be exported as attributes, there are no established conventions for the attribute key names to be used.


//similar.

Reply:
updated